src/Prometheus/PushGateway.php

```diff
@@ -55,12 +55,12 @@ public function delete($job, $groupingKey = null)
     }
 
     /**
-     * @param CollectorRegistry $collectorRegistry
+     * @param CollectorRegistry|null $collectorRegistry
      * @param $job
      * @param $groupingKey
      * @param $method
      */
-    private function doRequest(CollectorRegistry $collectorRegistry, $job, $groupingKey, $method)
+    private function doRequest(CollectorRegistry $collectorRegistry = null, $job, $groupingKey, $method)
```

Comment:
What argument errors does this prevent? When do they occur?

Reply:
When running the blackbox tests, I'm getting an argument error/notice since the `CollectorRegistry` is expected but `null` is given in `PushGateway::delete()` method.

Reply:
See #82.

```
There was 1 error:

1) Test\BlackBoxPushGatewayTest::pushGatewayShouldWork
Argument 1 passed to Prometheus\PushGateway::doRequest() must be an instance of Prometheus\CollectorRegistry, null given, called in /var/www/html/src/Prometheus/PushGateway.php on line 54 and defined

/var/www/html/src/Prometheus/PushGateway.php:63
/var/www/html/src/Prometheus/PushGateway.php:54
/var/www/html/tests/Test/BlackBoxPushGatewayTest.php:36
```